### ISSUE TITLE
Why is my minecraft crashing

### ISSUE THREAD

**Issue by AmmarDeeper34** [ddf8196/BetterRenderDragon#121]
hello so i downloaded better render dragon v1.4.4 the latest and I am having an issue when i run the Minecraft .bat then it opens Minecraft but it crashes and its happening everything and its say that There’s a problem with Minecraft. Reinstall the application from its original install location or contact your administrator. so what's happening here plz help me why is this happening in the installation guide i did everything it said me to do 
![image_2024-11-10_140305864](https://github.com/user-attachments/assets/9cf29752-a077-4166-bb66-4546a6152ef3)

**Comment by FlaredRoverCodes** [comment]
**Latest Version of Better Renderdragon Is Broken**

**Comment by AmmarDeeper34** [comment]
oh is what should i do should i intsall the older version of it

**Comment by faizul726** [comment]
did you install minecraft from somewhere else other than microsoft store?

**Comment by AmmarDeeper34** [comment]
yeah i used the github bedrock launcher as my bedrock launcher for launching bedrock or installing other versions

**Comment by AmmarDeeper34** [comment]
and also other thing is that my minecarft still does it even when i install 1.4.1 render dragon or which version will still work

**Comment by AmmarDeeper34** [comment]
> p

yes i did

**Comment by faizul726** [comment]
downgrading brd won't help @AmmarDeeper34  the issue is something else
